### ISSUE TITLE
How to define the order of actor execution within a single phase?

### ISSUE THREAD

**Issue by shruthidharani-4313** [oamg/leapp-repository#998]
I am adding a custom actor into RPMUpgrade phase and I know that there are many actors defined by default in the same phase. I need to run my actor at the last of RPMUpgrade phase and my actor class doesn't consume any input. How can I order actors within a Phase tag? How can I specify that the actor should run in postRPMUpgrade phase? I could see "RPMUpgradePhaseTag" available which is the main phase of rpm upgrade. As I am very new to this, I couldn't figure out or find any documentation on this. I could see the [inplace upgrade flow ](https://leapp.readthedocs.io/en/latest/_static/images/inplace-upgrade-workflow.svg) where I can see the order of Phases. It would be really helpful if someone helps me with this. Thanks in advance!!

**Comment by pirat89** [comment]
Hi @shruthidharani-4313, 

* tl;dr; specify the `After` stage of the phase in `tags` as [here](https://github.com/oamg/leapp-repository/blob/1859d1811d6331eda8c9684fac47b12ca2e796ae/repos/system_upgrade/common/actors/removeresumeservice/actor.py#L21)

It's not possible to affect the order of actors execution inside a single phase except by consume/produce messages. However, we have covered your particular use-case when people need to execute a custom acter `Before` or `After` a 
phase. For that purpose each phase execution has 3 [stages](https://leapp.readthedocs.io/en/latest/terminology.html?highlight=phase#stage) implemented by leapp (that's why each phase has 3 columns in the diagram). You can think about those stages as virtual phases if it helps. So you can specify in tags e.g. `PhaseTag.Before` or `PhaseTag.After` to execute the actor in a specific phase.

In your case  your case you can specify in tags `RPMUpgradePhaseTag.After` and the actor will be executed during the `After` stage of the particular phase. You can see this example for a different phase: https://github.com/oamg/leapp-repository/blob/1859d1811d6331eda8c9684fac47b12ca2e796ae/repos/system_upgrade/common/actors/removeresumeservice/actor.py#L21
